src/modules/logbook/stores/current-logbook-date-store.js

```diff
@@ -1,12 +1,13 @@
 import nuclearJS from 'nuclear-js';
-import dateToStr from '../../../util/date-to-str';
```

Comment:
We can remove this file completely from Home Assistant JS, it is no longer used.